template/FullStack/Vite(Frontend)+Flask(Backend)/server/utils/Auth/auth.py

```
@@ -0,0 +1,44 @@
+from flask import request, jsonify
+from models import User  
+from werkzeug.security import generate_password_hash, check_password_hash 
+from bson import ObjectId 
```

Ruff

`bson.ObjectId` imported but unused

Remove unused import: `bson.ObjectId`

(F401)